doc/interfaces/gibbs_unringing_flow.rst

```diff
@@ -32,12 +32,11 @@ ringing-free volume::
 
 To run the Gibbs unringing method, we need to specify the path to the input
 data. This path may contain wildcards to process multiple inputs at once.
-You can also specify the optional arguments. In this case, we will be
-specifying the number of threads (``num_threads``) and output directory
-(``out_dir``). The number of threads allows to exploit the available
-computational power and accelerate the processing. The maximum number of
-threads available depends on the CPU of the computer, so users are expected
-to set an appropriate value based on their platform.
```

Comment:
I think this was written this way having in mind that part of the workflows audience/users might not be that familiar with what threads are, what they represent or how a value that fits in their case can be determined. It does not aim to be a thorough explanation, but I found it helpful.

I think re-using it below could be nice.

Also, although not written in stone, note that we try to wrap at 79 characters.

Reply:
I found the tutorial a bit lengthy but I see your point about the broader audience, let's keep it.